RFC/src/RFC-0241_AtomicSwapXMR.md

```diff
@@ -0,0 +1,238 @@
+and the [commit team](https://github.com/comit-network/xmr-btc-swap). Due to the way that TariScript works we have a few advantages over bitcoin script when it comes to [adaptor signatures](https://tlu.tarilabs.com/cryptography/introduction-to-scriptless-scripts#adaptor-signatures) as the [script key] was spesifically designed with [scriptless scripts](https://tlu.tarilabs.com/cryptography/introduction-to-scriptless-scripts).
+
+### Method
+
+The basic happy outline of a Tari - Monero atomic swap is described here and more detail will follow. We will assume here that Alice wants to trade her XTR for Bob's XMR.
+
+* Negotiation - Here both parties negotiate about the values, and how the Monero and Tari Utxo's will look
+* Commitment - Here both parties commit to their use of keys
+* XTR payment - Here the XTR payment is made to a multi party UTXO containing a script
+* XMR Payment - Here the Monero payment is made to a multiparty [scriptless script](https://tlu.tarilabs.com/cryptography/introduction-to-scriptless-scripts) utxo.
+* Claim XTR - Here the XTR is claim and in claiming it, with the help of an , the XMR private key is revealed
+* Claim XMR - Here the XMR is claimed with the revealed key.
+
+Please take note of the notation used in [TariScript] and specifically notation used on the signatures on the [transaction inputs](RFC-0201_TariScript.md##transaction-input-changes) , other notation will be noted in the [Notation](#notation) section.
+
+### TariScript
+
+The Script used for the Tari UTXO is as follows:
+``` TariScript,ignore
+   CheckHeight(height)
+   LtZero
+   IFTHEN
+      PushPubkey(K_{Ss})
+   Else
+      PushPubkey(K_{Sr})
+   ENDIF
+```
+
+### Negotiation
+
+Alice and Bob have to negotiate about the rate of exchange, and the amount to be exchanged in the atomic swap. 
+They also need to decide how the two UTXO's will look on the block chain. In order to do so the following needs to be finalized:
+
+* Amount of Tari to swap for the amount of Monero
+* Monero public key parts \\(X_a\\), \\(X_b\\) and its aggregate form \\(X\\)
+* Tari [script key] parts \\(K_Ssa\\), \\(K_Ssb\\) and its aggregate form \\(K_Ss\\)
+* Tari [script key] parts \\(K_Sra\\), \\(K_Srb\\) and its aggregate form \\(K_Sr\\)
+* All of the nonces used in the script signature creation for \\(K_Ss\\): \\(r_{Ssa_a}, r_{Ssa_b}, r_{Ssb_a, r_{Ssb_b}}\\), where \\(R_{Ssa}, R_{Ssb}\\) is respectively for Alice and Bob's nonces
+* All of the nonces used in the script signature creation for \\(K_Sr\\): \\(r_{Sra_a}, r_{Sra_b}, r_{Srb_a, r_{Srb_b}}\\), where \\(R_{Sra}, R_{Srb}\\) is respectively for Alice and Bob's nonces
+* The [TariScript] to be used in the Tari UTXO
+* The blinding factor \\(k_i\\) for the Tari UTXO
+
+
+### Commitment phase
+
+This phase allows Alice and Bob to commit to the use of their keys. 
+Alice needs to provide bob the following:
+
+* Adaptor signature \\(b'_{Sra}\\) for \\(b_{Sra}\\)
+* Signature \\(a_{Sra}\\)
```

Comment:
According to the original paper, yes. But I am sticking to the notation we used in the TariScript RFC: https://rfc.tari.com/RFC-0201_TariScript.html#transaction-output-changes
Here we used `a, b` because `v` is the symbol used for value in all MW documentation.

Reply:
Ok sounds good